DataProc/CPass0/main_recCPass0.C

```diff
@@ -109,6 +109,9 @@ void main_recCPass0(const char *filename="raw.root",Int_t nevents=-1, const char
   else {
     // setup ocdb by custom (if any) or default settings
     if (gSystem->AccessPathName("localOCDBaccessConfig.C", kFileExists)==0) {        
+      TString envRunNumBuf = gSystem->Getenv("ALIEN_JDL_LPMRUNNUMBER");
```

Comment:
Executed only if `localOCDBaccessConfig.C` is there, which is not the case on the Grid.